tests/unit/UI/Alert/DialogBox.spec.js

```diff
@@ -0,0 +1,29 @@
+import { mount } from "@vue/test-utils";
```

Comment:
I think these should follow the same folder structure starting from `src` right? So, there should be a `components` folder within which the `UI` folder should be there, right?

Reply:
aah okay. got it.